### PR TITLE
[SDA-7890] Adjust resources to match /oidc_configs endpoints

### PR DESCRIPTION
**Pull request by gdbranco**

### CODE REVIEW

File: model/clusters_mgmt/v1/oidc_configs_resource.model
Comment (hunterkepley) [inline model/clusters_mgmt/v1/oidc_configs_resource.model:1]: For these comments, should they still include `hosted`?

Comment (hunterkepley) [reply model/clusters_mgmt/v1/oidc_configs_resource.model:1]: for the top 2 comments that is

File: model/clusters_mgmt/v1/oidc_config_resource.model [hunk 1]
```diff
@@ -15,13 +15,18 @@ limitations under the License.
 */
 
 // Manages a hosted oidc configuration.
```
Comment (hunterkepley) [inline model/clusters_mgmt/v1/oidc_config_resource.model:17]: Is this still always a `hosted` oidc configuration?

Comment (gdbranco) [reply model/clusters_mgmt/v1/oidc_config_resource.model:17]: Oh missed it somewhere

**Pull request merged**